python/ray/serve/tests/test_autoscaling_policy.py

```diff
@@ -316,7 +316,11 @@ def check_running():
     start = time.time()
     result = handle.remote().result()
     cold_start_time = time.time() - start
-    assert cold_start_time < 3
+    if sys.platform == "win32":
+        timeout = 10  # Windows has a longer tail.
```

Comment:
Non-blocker: I feel maybe we should tie up this a bit to maybe 4s just in case it hides some significant issues, but will defer to what others think :)

Reply:
hmm. I changed to 5.

Reply:
5 sounds good to me.